pkg/codegen/nodejs/gen.go

```diff
@@ -2143,6 +2143,10 @@ func genNPMPackageMetadata(pkg *schema.Package, info NodePackageInfo) string {
 		},
 	}
 
+	if pkg.Version != nil && info.RespectSchemaVersion {
+		npminfo.Version = pkg.Version.String()
```

Comment:
Shouldn't we also specify the version passed to the install script on line 2135 rather than `${VERSION}`?

Also, possibly as a separate change, I'm wondering if we should update the install script to exit early if the version argument has either `${VERSION}` _or_ `0.0.0`. The first customer to use `respectSchemaVersion` will be setting it to `0.0.0` so that the generated SDK will have a valid version of 0.0.0 in `package.json` so that local npm install/link can be done using the directory containing the generated SDK (rather than having to do an additional build step and then do the local npm install/link from a separate bin directory).